product_docs/docs/tde/15/secure_key/disabling_key.mdx

```diff
@@ -1,9 +1,9 @@
 ---
 title: Disabling the key wrapping
-description: Learn how skip using a wrapping key.
+description: Learn how to prevent using a wrapping key.
 deepToc: true
 ---
 
 If you don't want key wrapping, for example for testing, then you must set the wrap and unwrap commands to the special value `-`.
 
-This setting specifies to use the key from the file without further processing. This approach differs from not setting a wrap or unwrap command at all, and from setting either/both to an empty string. Having no wrap or unwrap command set when transparent data encryption is used leaves your data encryption key unsecured, and results in a fatal error when running an affected utility program.
+This setting specifies to use the key from the file without further processing. This approach differs from not setting a wrap or unwrap command at all and from setting either or both to an empty string. Having no wrap or unwrap command set when transparent data encryption is used leaves your data encryption key unsecured and results in a fatal error when running an affected utility program.
```

Comment:
```suggestion
This setting specifies to use the key from the file without further processing. This approach differs from not setting a wrap or unwrap command at all and from setting either or both to an empty string. Having no wrap or unwrap command set when TDE is used leaves your data encryption key unsecured and results in a fatal error when running an affected utility program.
```

Reply:
Really, we should use TDE when referring to the product because  transparent data encryption is a technology, and it's confusing whether you are referring to the product or the technology when it's spelled out. Maybe we should be referring to it as EDB TDE when we are referring to the product?